app/backend/user/models.py

```
@@ -55,12 +56,30 @@ def create_superuser(
+        extra_fields.setdefault(
+            "is_legacy",
+            True,
+        )
+        if "dru" not in extra_fields:
+            try:
+                extra_fields["dru"] = DRU.objects.get(id=1)
+            except DRU.DoesNotExist:
+                raise ValueError(
+                    "create_superuser requires a DRU instance; "
+                    "Please run the seeders to create a default DRU."
+                )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)